applications/nrf5340_audio/src/audio/streamctrl.c

```diff
@@ -351,9 +353,10 @@ static void le_audio_evt_handler(enum le_audio_evt_type event)
 
 		audio_system_start();
 		stream_state_set(STATE_STREAMING);
-		ret = led_blink(LED_APP_1_BLUE);
-		ERR_CHK(ret);
-
+		if (CONFIG_AUDIO_DEV == GATEWAY) {
+			ret = led_blink(LED_APP_1_BLUE);
```

Comment:
How much more work would it be to have the sync blink on the gateway as well? This can blink on the same SEQ_NR as the headsets. The gateway LED would in such a case change state before the headsets, but that will be preferable compared to now when the gateway blink will "appear to drift" compared to the headsets.

Reply:
The sync LEDS (headset) are now blinking on a different time period than the GW to prevent misconception about the LED "appear to drift".

Reply:
Great.